src/Routing/Router.php

```diff
@@ -216,7 +216,7 @@ public function addExistingRoutes(IlluminateRouteCollection $routes)
      */
     public function dispatch(Request $request, $morph = true)
     {
-        if (! $this->isApiRequest($request)) {
+        if (! $this->isApiRequest($request) OR $request->method() == 'OPTIONS') {
```

Comment:
@lucasmichot true. An artefact of an earlier idea, where I was handing the 'OPTIONS' response elsewhere based on the api request and not just handing any 'OPTIONS' request off to Laravel. Branch has been updated.